slave/executors.py

```diff
@@ -84,16 +85,18 @@ def wait_for_results(self, timeout):
                 suite = " when running {}".format(self.benchmark.suite)
             except:
                 pass
-            print "TIMEOUT (executor){}".format(suite)
+            logger = logging.getLogger('BrowserExecutor.wait_for_results')
+            utils.log_error(logger, "TIMEOUT (executor){}".format(suite))
```

Comment:
preexisting nit: can you add a space between closing parens and `{}` please?

Reply:
ok.